components/README.md

```diff
@@ -0,0 +1,141 @@
+[TOC]
+
+
+
+# Dialogs
+
+Dialogs are a standard way to inform the user about some result, giving the user option to react to it.
+
+<details>
+    <summary>Code snippet</summary>
+    <br>
+https://github.com/appunite/Loudius/blob/8e616a9a05e668d11425a40c9f452eb3091dcc2a/components/src/main/java/com/appunite/loudius/components/components/LoudiusDialog.kt#L25-L48
+</details>
+
+### Standard
```

Comment:
suggestion: currently those images takes a lot of space. Maybe it would be worth training to put them side-by-side by using a table widget?

Reply:
sure, good idea